content/news/release-20180217.md

```diff
@@ -0,0 +1,59 @@
+We are excited to announce the first [OpenRA release](/download/) of this year!
+Over 700 commits from 43 contributors were merged over the last six months, with each commit helping to fix bugs, improve code quality, or add new features that improve the single- and multi-player experience.
+
+The main focus of this development cycle was modernizing core parts of the engine and game code to support the work-in-progress Tiberian Sun mod and other community projects. These shouldn't have visibly changed our released mods, but big changes always have a risk of introducing unexpected bugs or side-effects. Please note that changes to the way hotkeys are managed mean you will need to redefine any custom hotkeys in the Input settings.
```

Comment:
Better to remove "These shouldn't have visibly changed our released mods, but big changes always have a risk of introducing unexpected bugs or side-effects" - this isn't the playtest anymore.

Reply:
That sentence could be replaced with something thanking testers for finding bugs?